Godeps/_workspace/src/github.com/steakknife/hamming/README.md

```diff
@@ -0,0 +1,3 @@
+Copyright (c) 2014 Barry Allard
+
+MIT license
```

Comment:
the code doesn't bear an actual `LICENSE` file. https://github.com/steakknife/hamming/pull/3